lang/en/tool_excimer.php

```diff
@@ -83,7 +83,7 @@
 $string['field_group'] = 'Group';
 $string['field_created'] = 'Created';
 $string['field_finished'] = 'Finished';
-$string['field_user'] = 'User';
+$string['field_userid'] = 'User';
```

Comment:
But field_user is currently being used in other places too (e.g. mustache template)? I think this should be an additional string, not a replacement.

See below:
```php
# admin/tool/excimer/templates/flamegraph.mustache
                    <th>{{#str}} field_user, tool_excimer {{/str}}</th>
                    <th>{{#str}} field_useragent, tool_excimer {{/str}}</th>
# admin/tool/excimer/classes/privacy/provider.php
                'userid' => 'field_userid',
                'useragent' => 'field_useragent',
# admin/tool/excimer/lang/en/tool_excimer.php
$string['field_user'] = 'User';
$string['field_useragent'] = 'User agent';
```

Reply:
The field used to be called user, but changed to userid. The mustache template refers to the same thing, so it was changed.
A quick global search revealed no more instances of 'field_user'.

Reply:
Yeah, the problem is not all the fields are set as the full string name, and are used in dynamic strings which makes these sorts of things harder to detect from a grep search.

```
Invalid get_string() identifier: 'field_user' or component 'tool_excimer'. Perhaps you are missing $string['field_user'] = ''; in /var/www/39/admin/tool/excimer/lang/en/tool_excimer.php?

    line 353 of /lib/classes/string_manager_standard.php: call to debugging()
    line 7072 of /lib/moodlelib.php: call to core_string_manager_standard->get_string()
    line 73 of /admin/tool/excimer/classes/profile_table.php: call to get_string()
    line 47 of /admin/tool/excimer/classes/profile_table_page.php: call to tool_excimer\profile_table->make_columns()
    line 51 of /admin/tool/excimer/slowest.php: call to tool_excimer\profile_table_page::display()
```
The offending section is at the table class, e.g. (`admin/tool/excimer/classes/profile_table.php`)